bin/pyshepseg_runtests.py

```diff
@@ -337,6 +373,41 @@ def checkSegmentation(imgfile, segfile, meanColNames, stdColNames):
     return pcntMatch
 
 
+def checkSpatialColumns(segfile, eastingCol, northingCol):
+    """
+    Do a quick check of eastingCol and northingCol which were
+    calculated using calcPerSegmentSpatialStatsTiled().
+    
+    Returns True if the calculated coordinates are the same
+    as the actual coordinates (within a tolerance).
+    
+    """
+    # read in the data to check
+    eastingData = readColumn(segfile, eastingCol)
+    northingData = readColumn(segfile, northingCol)
+
+    # read in the segfile
+    seg = readSeg(segfile)
+    # work out the sizes of the segments (needed for makeSegmentLocations)
+    segSize = shepseg.makeSegSize(seg)
+    TOL = 0.2
```

Comment:
The tolerance is larger than I would have expected. Did you find it needed this much, or was that just an initial guess?

Reply:
Was just a random number. Reduced to 0.01 and all still seems fine. Should I go down further?

Reply:
It should be as small as reasonable. If we made a change which broke the spatial stuff in some small way (e.g. missed a few pixels in each segment), then having too large a tolerance would allow it to slip through. 

I suggest running a test and holding on to the largest difference which actually appears. Then round this up to a single significant digit (just for neatness), and use that. Because the test data is always the same, this should be fine. 

The coordinates of each pixel are whole numbers, with only 4 significant digits, the total range of values is less than 10000, and the calculations are all carried out as float64, so my guess is that TOL could be quite a lot smaller. However, best to check.

Reply:
Good point. largest difference for me is 0.00024406740521953907. Should we go with 0.0002 ?

Reply:
Sorry I meant 0.0003

Reply:
:-)
Yes, that sounds perfect. Very good.